### ISSUE TITLE
Swagger: Model text description is repeated. Example added

### ISSUE THREAD

**Issue by Trudeaucj** [concur/developer.concur.com#370]
The description on the model for objects is repeated. Text needs to be updated to correct description. 

https://developer.concur.com/api-explorer/v3-0/QuickExpenses.html

QuickExpensePost {
Comment (string, optional): The descriptive text for the vendor for the quick expense. This often matches the Merchant Name found in a credit card transaction. Max Length: 64 ,
CurrencyCode (string, optional): The descriptive text for the vendor for the quick expense. This often matches the Merchant Name found in a credit card transaction. Max Length: 64 ,
ExpenseTypeCode (string, optional): The descriptive text for the vendor for the quick expense. This often matches the Merchant Name found in a credit card transaction. Max Length: 64 ,
LocationCity (string, optional): The descriptive text for the vendor for the quick expense. This often matches the Merchant Name found in a credit card transaction. Max Length: 64 ,
LocationCountry (string, optional): The descriptive text for the vendor for the quick expense. This often matches the Merchant Name found in a credit card transaction. Max Length: 64 ,

**Comment by henryyamamoto-concur** [comment]
Yup. It looks OK in the static I pulled from old swagger: https://developer.concur.com/api-reference/expense/quick-expense/index.html

**Comment by henryyamamoto-concur** [comment]
@howarddierking That's weird. I thought we pulled these directly from CTE.

**Comment by howarddierking** [comment]
we did - not sure how this happened, but nice that it's simple to correct.

**Comment by Trudeaucj** [comment]
:shipit:

**Comment by merittrompeter** [comment]
I've edited the text to remove the repeated description: "The descriptive text for the vendor for the quick expense. This often matches the Merchant Name found in a credit card transaction. Max Length: 64".

However, there are a few places where I removed it (because it didn't make sense), but I could not find a suitable replacement description on the old swagger. 

Those fields are: 
- definitions ID
- definitions URI
- Quick Expense Post Location Subdivision
- Quick Expense Put Location Subdivision
- Quick Expense Post SpendCategory Code
- Quick Expense Put SpendCategory Code
